client/components/tinymce/plugins/simple-payments/dialog/product-image-picker.jsx

```diff
@@ -131,11 +126,8 @@ class ProductImagePicker extends Component {
 	}
 }
 
-export default connect( ( state, ownProps ) => {
-	const siteId = getSelectedSiteId( state );
-
+export default connect( state => {
```

Comment:
Needless nitpick: we could simplify this into

`connect( state => ( { siteId: getSelectedSiteId( state ) } ) )`

Reply:
Updated in 6b9abf9be28e332e8849e04833d95496c959fd42.